### PR TITLE
hotfix: enable minting on rmrk2

### LINKED ISSUE #5646: Enable Classic minting on RMRK2
### What happened?

Once #5631 is merged it is urgent to enable minting on RMRK2 (/ksm)

@Jarsen136 already made a similar PR #5630 but then he closed it 

### Please reproduce in steps

1. Open /ksm
2. Do not see ability to create > classic

### Expected Behavior

1. Open /ksm
2. See create > classic option in the navbar
3. Try to create collection 
4. send tx hash

### What browsers are you seeing the problem on?

_No response_

### At which address did you encounter bug?

_No response_

### Are you logged in?

None

### Which wallet you are using?

_No response_

### Screenshots

Screenshot 2023-04-13 at 12 50 50

### Relevant log output

_No response_

### Payment link for reward

https://kodadot.xyz/transfer/?target=%3CMy_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address%3E

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] closed #5646

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: utils/mixins/rmrkVersionMixin.ts [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
+
 /*
  * refer to https://stackoverflow.com/questions/51873087/unable-to-use-mixins-in-vue-with-typescript
  * usage import Component, { mixins } from 'vue-class-component';
  * class ExtendedClass extends mixins(SubscribeMixin) {
  */
 @Component
-export default class RmrkVersionMixin extends Vue {
-  public version = '1.0.0'
+export default class RmrkVersionMixin extends mixins(PrefixMixin) {
```
Comment (vikiival) [inline utils/mixins/rmrkVersionMixin.ts:10]: OK, btw I used different approach, 

```
composables/useRmrkVersion.ts
```

but this is also lgtm for me

File: pages/ksm/mint.vue
Comment (vikiival) [inline pages/ksm/mint.vue:1]: Please no Simple mint yet, we do not have logic for that

Comment (Jarsen136) [reply pages/ksm/mint.vue:1]: Thanks. I have updated it.

**Review by vikiival — APPROVED** [review]

File: utils/mixins/rmrkVersionMixin.ts
[the remark this answers is not in the document]
Comment (Jarsen136) [reply utils/mixins/rmrkVersionMixin.ts:1]: ok, I have brought it back though it was not to be used for now.

**Review by vikiival — APPROVED** [review]

**Review by daiagi — APPROVED** [review]
lgtm

**Comment by vikiival** [comment]
Guess what

Screenshot 2023-04-14 at 14 17 29

**Comment by vikiival** [comment]
@Jarsen136 I pushed new minimark that enables this

- https://github.com/kodadot/packages/issues/126

**Review by daiagi — APPROVED** [review]

File: composables/transaction/mintToken/transactionMintRmrk.ts [hunk 2]
```diff
@@ -23,27 +27,34 @@ export async function execMintRmrk(
...
 
   const { id: collectionId, alreadyMinted: collectionAlreadyMinted } = item
     .token.selectedCollection as MintedCollectionKusama
   const { edition, name, postfix } = item.token
 
   const metadata = await constructMeta(item, { enableCarbonOffset: true })
-
-  const mint = createMultipleNFT(
+  const updateNameFn = postfix && edition > 1 ? basicUpdateFunction : undefined
+  const createMutipleNFTFn = isV2.value ? createMultipleItem : createMultipleNFT
```
Comment (vikiival) [inline composables/transaction/mintToken/transactionMintRmrk.ts:38]:
```suggestion
  const mintFunction = isV2.value ? createMultipleItem : createMultipleNFT
```

**Comment by Jarsen136** [comment]
> @Jarsen136 I pushed new minimark that enables this
> 
> * [`createMultipleNFT` is missing in v2 packages#126](https://github.com/kodadot/packages/issues/126)

I have imported something from minimark v2 to try to make it work, but it seems that minting nft token process still failed. Could we leave it to another issue #5647 ?

File: composables/transaction/mintToken/transactionMintRmrk.ts (hunk 2 above)
Comment (Jarsen136) [reply composables/transaction/mintToken/transactionMintRmrk.ts]: Done

**Comment by vikiival** [comment]
> Could we leave it to another issue https://github.com/kodadot/nft-gallery/issues/5647 

Sure just please hide NFT tab for now

**Comment by vikiival** [comment]
I managed 

https://singular.app/collectibles/kusama/8CC1B91E899D9BE40E-KDWF/17482116-8CC1B91E899D9BE40E-KDWF-KODAWAIFU_1-00000001

https://deploy-preview-5630--koda-canary.netlify.app/ksm/gallery/17482116-8CC1B91E899D9BE40E-KDWF-KODAWAIFU_1-00000001

**Comment by vikiival** [comment]
pay 35 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $35 @ 37.34 USD/KSM ~ 0.937 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0xc9fd30ff04c69989c9b4cb0fd3a512797e4ad72ec1cf46734c579767bc22e932](https://kusama.subscan.io/extrinsic/0xc9fd30ff04c69989c9b4cb0fd3a512797e4ad72ec1cf46734c579767bc22e932)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**